app/views/static_pages/privacy_policy.html.erb

```diff
@@ -112,14 +112,14 @@
 
     <p class="govuk-body">
       Neither DfE nor its delivery partners will keep your personal data longer than we need it for the purpose(s) of
-      research, security and/or delivery of the service, and in any case not longer than XXXX years. Your data will be
-      deleted when DfE and its delivery partners no longer need it for these purposes, or when XXXX years have passed,
+      research, security and/or delivery of the service, and in any case not longer than 7 years. Your data will be
+      deleted when DfE and its delivery partners no longer need it for these purposes, or when 7 years have passed,
       whichever comes first.
     </p>
 
     <p class="govuk-body">
-      Partial IP addresses will be deleted after XXXX days unless their request caused an error, in which case they remain
-      in our logs for XXXX days.
+      Partial IP addresses will be deleted after 7 days unless their request caused an error, in which case they remain
+      in our logs for 30 days.
```

Comment:
IIRC, TVS did something clever with munging the IPs to they only stored partial ones, but I don't remember the details. We don't store any IPs at all currently, and we're not set up to make this statement at all true if we did start to collect them. Can we just remove it?